WebDriverAgentLib/Vendor/YYCache/YYCache.m

```diff
@@ -132,8 +132,8 @@ - (void)removeAllObjectsWithProgressBlock:(void(^)(int removedCount, int totalCo
 }
 
 - (NSString *)description {
-    if (_name) return [NSString stringWithFormat:@"<%@: %p> (%@)", self.class, self, _name];
-    else return [NSString stringWithFormat:@"<%@: %p>", self.class, self];
+  if (_name) return [NSString stringWithFormat:@"<%@: %@> (%@)", self.class, self, _name];
```

Comment:
Please keep the format specifier. Simply cast `self` to `(void *)self` in order to get rid of the waning